doc/sphinx/.sphinx/spellingcheck.yaml

```
@@ -9,7 +9,7 @@ matrix:
     - .custom_wordlist.txt
     output: .sphinx/.wordlist.dic
   sources:
-  - _build/**/*.html
+  - _build/**/*.html|!_build/**/api/**/*.html
```

The api is directly under `_build`, no?

```suggestion
  - _build/**/*.html|!_build/api/**/*.html
```